src/solidity-frontend/solidity_convert.cpp

```diff
@@ -2030,3 +2052,11 @@ bool solidity_convertert::is_dyn_array(const nlohmann::json &json_in)
   }
   return false;
 }
+
+void solidity_convertert::print_json(nlohmann::json &json) {
+  std::cout << std::setw(2) << json << std::endl;
```

Comment:
If you don't want to pass down the messaget object. You can use the `default_message` for it (since this is only a dump!).